llvm/lib/Target/NVPTX/NVPTXSubtarget.h

```diff
@@ -59,42 +59,55 @@ class NVPTXSubtarget : public NVPTXGenSubtargetInfo {
   NVPTXSubtarget(const Triple &TT, const std::string &CPU,
                  const std::string &FS, const NVPTXTargetMachine &TM);
 
-  const TargetFrameLowering *getFrameLowering() const override {
+  inline const TargetFrameLowering *getFrameLowering() const override {
```

Comment:
adding `inline` is unnecessary here, please remove.

https://en.cppreference.com/w/cpp/language/inline#:~:text=A%20function%20defined%20entirely%20inside,(since%20C%2B%2B20).

> A function defined entirely inside a [class/struct/union definition](https://en.cppreference.com/w/cpp/language/classes), whether it's a member function or a non-member friend function, is implicitly an inline function